### ISSUE TITLE
cedar-14 end-of-life

### ISSUE THREAD

**Issue by ajschumacher** [HarlanH/code-for-dc-edu#95]
Somehow I'm the point-of-contact for http://code-for-dc-edu.herokuapp.com/ and it's on the cedar-14 stack on heroku, which disappears in May or so... I tried the quick-and-easy upgrade to heroku-18 but it fails with "This version of npm (1.3.11) has several known issues - consider upgrading to the latest release (6.14.3)" and I can't even with the node/npm craziness... I'm going to let it just die unless somebody wants to take the lead on upgrading it, in which case I'll try to help with getting control of the heroku app moved over to whoever else, but I think this project is pretty dead, yeah?

**Comment by HarlanH** [comment]
Yeah, I think 7 years or whatever it was is a reasonable lifetime for this
project :)
